action.yml

```diff
@@ -20,6 +20,13 @@ inputs:
   generate_attestations:
     description: "Whether to generate artifact attestations for release binaries to establish build provenance, defaults to `false` if unspecified"
     default: false
+  release_android:
+    description: "Whether to release android binaries, defaults to `false` if unspecified"
+    default: false
+  android_ndk_home:
+    description: "Path to the Android NDK for android-amd64 and android-arm64 builds, required for Go 1.22 or newer, defaults to `ANDROID_NDK_HOME` environment variable if unspecified"
+  android_sdk_version:
+    description: "Android SDK build-tools version"
```

Comment:
I think these changes deserve an update to `README.md` explaining why we are going to be cutting a `v2` update, how extension authors choosing to support Android / Termux community need to understand which Android SDK version they are targeting.

Reply:
Also, I wonder if we should update README that if you need to customize the process, then you might be on your own to copying and modifying the `build_and_release.sh` file for your needs.

Reply:
I added some additional details to the readme for your review 🙇